ironfish/src/wallet/walletdb/walletdb.ts

```diff
@@ -385,6 +386,19 @@ export class WalletDB {
     await this.nonChainNoteHashes.clear(tx, account.prefixRange)
   }
 
+  async *loadSequenceToTransactionHashes(
```

Comment:
We already have this added by hugh

Reply:
Hugh's method takes a sequence range. I'll leave this as is